### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by Mark-Adaptive**
Can you see if this works on your end? I am still unable to fetch any data from the API on my end.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -88,6 +114,10 @@ const dataStore = producerMiddleWare(
 function App() {
   useEffect(() => {
     dataStore.dispatch({ type: "searchSymbol", payload: INITIAL_STOCK });
+    /*dataStore.dispatch({
```
Comment (jrothweiler) [inline src/App.js:117]: Ok, looks like the index fetch behavior is opt in, which is good

File: src/components/generics/displayWrapper.js [hunk 2]
```diff
@@ -17,7 +17,6 @@ export const DisplayWrapper = styled("div")(
         display: "flex",
         flexDirection: "row",
         flexWrap: "wrap",
-        justifyContent: "space-between",
```
Comment (jrothweiler) [inline src/components/generics/displayWrapper.js:20]: why was this line removed? I feel like removing it could be problematic as some of our flex rows assume it is space-between

File: src/stockTrader.js [hunk 3]
```diff
@@ -41,6 +47,9 @@ export const StockTrader = () => {
           <TopPeers height="15%" />
         </DisplayWrapper>
       </DisplayWrapper>
+      <DisplayWrapper mt="2.0rem" width="100%" variant="flexRow">
```
Comment (jrothweiler) [inline src/stockTrader.js:50]: Do we need flexRow if there is only one thing in this row? Is width: 100% sufficient?

File: src/components/generics/displayWrapper.js (hunk 2 above)
Comment (Mark-Adaptive) [reply src/components/generics/displayWrapper.js:20]: I removed it because space between was not always the desired behavior in every flexRow. For instance in the footer I actually don't want space between to be active on the indexes. I added justifyContent: space-between separately in all areas that previously needed that behavior so everything should still look correct.

File: src/stockTrader.js (hunk 3 above)
Comment (Mark-Adaptive) [reply src/stockTrader.js:50]: You're right, don't need flexRow there although we might if we also add the favorites section. I removed it for now.

File: server/index.js [hunk 4]
```diff
@@ -56,20 +56,20 @@ async function getQuoteData(symbol) {
...
-    week52Low: safeToFixed(week52Low, 2),
-    high: safeToFixed(high, 2),
-    low: safeToFixed(low, 2),
-    latestPrice: safeToFixed(latestPrice, 2),
+    symbol: quoteData.symbol,
+    previousClose: previousClose.toFixed(2),
+    week52High: week52High.toFixed(2),
+    week52Low: week52Low.toFixed(2),
+    high: high.toFixed(2),
+    low: low.toFixed(2),
```
Comment (jrothweiler) [inline server/index.js:64]: These should still use `safeToFixed`
